### PR TITLE
CORE-8331 Add static ids for "Select Metadata Template" dialog.

### PR DESCRIPTION
**Pull request by sri2k1us**

### CODE REVIEW

File: de-lib/src/main/java/org/iplantc/de/diskResource/client/views/metadata/dialogs/SelectMetadataTemplateDialog.java [hunk 1]
```diff
@@ -52,8 +52,15 @@
 
     private MetadataView.Presenter.Appearance appearance;
 
+    private String BASE_ID = "select_metadata_template";
```
Comment (ash3rz) [inline de-lib/src/main/java/org/iplantc/de/diskResource/client/views/metadata/dialogs/SelectMetadataTemplateDialog.java]: I think for consistency these IDs should probably be defined with the rest of the [Metadata ids](https://github.com/cyverse-de/ui/blob/f69465f48b2a62b5a4a179e0fc5c1006f884eec4/de-lib/src/main/java/org/iplantc/de/diskResource/share/DiskResourceModule.java) and should be camel case.

Comment (sri2k1us) [inline de-lib/src/main/java/org/iplantc/de/diskResource/client/views/metadata/dialogs/SelectMetadataTemplateDialog.java]: Also found close tool button is not getting a static id plus throws a NPE 👎

Comment (ash3rz) [inline de-lib/src/main/java/org/iplantc/de/diskResource/client/views/metadata/dialogs/SelectMetadataTemplateDialog.java]: I've found that certain elements can't get static IDs until after they've been rendered (after the show() method has been called) which might be what you're noticing?

Comment (sri2k1us) [inline de-lib/src/main/java/org/iplantc/de/diskResource/client/views/metadata/dialogs/SelectMetadataTemplateDialog.java]: 👍

Comment (sri2k1us) [inline de-lib/src/main/java/org/iplantc/de/diskResource/client/views/metadata/dialogs/SelectMetadataTemplateDialog.java]: Commited changes according to review.

**Comment by ash3rz** [comment]
LGTM 🚢

**Comment by sri2k1us** [comment]
👍

**Pull request merged**